api/go.mod

```diff
@@ -14,3 +14,5 @@ require (
 	sigs.k8s.io/kustomize/kyaml v0.13.3
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace sigs.k8s.io/kustomize/kyaml => ../kyaml
```

Comment:
clean up?

Reply:
This is necessary while in development mode. We usually make this change after every release, I must have forgotten to do it after the last release

Reply:
oh oh I see. Basically the api and other kyaml/libs (kio, fn) are always tied but released separately.